tests/parse.spec.js

```diff
@@ -232,4 +232,54 @@ describe('Parse', function() {
     ]);
   });
 
+  it('should handle unknown escape sequences inside a value', function() {
+    m.chai.expect(parse([
+      'device: /dev/disk1',
+      'description: "\\]=-01`234567890=-098e"'
+    ].join('\n'))).to.deep.equal([
+      {
+        device: '/dev/disk1',
+        description: ']=-01`234567890=-098e'
+      }
+    ]);
+  });
+
+  it('should delete known escape sequences', function() {
+    m.chai.expect(parse([
+      'device: /dev/disk1',
+      'description: "foo\bbar\rbaz"'
+    ].join('\n'))).to.deep.equal([
+      {
+        device: '/dev/disk1',
+        description: 'foobarbaz'
+      }
+    ]);
+  });
+
+  it('should not remove tabs inside values', function() {
```

Comment:
Huh? Seems like it's actually replacing tabs by 4 spaces. If that _is_ the intended behaviour, this test should probably be renamed ;-)